script/apply-patches

```diff
@@ -52,7 +52,7 @@ def apply_patches_for_dir(directory):
 def parse_args():
   parser = argparse.ArgumentParser(description='Apply all required patches.')
 
-  parser.add_argument('-t', '--target-arch',
+  parser.add_argument('-t', '--target_arch',
```

Comment:
this looks like a breaking change, but probably nbd. if you wanted to be extra-cautious, you could:

```python
parser.add_argument('--target-arch', help=argparse.SUPPRESS)
```

Reply:
I hope that nobody actually calls `apply-patches` manually.
Since the change doesn't break the CI build, we should be fine.

Reply:
> idea for a backwards-compatible option

The arg [was added](https://github.com/electron/libchromiumcontent/pull/470/commits/931f976528e634600048a79787b071a6c9cbc7c3) only three weeks ago, nobody will ever notice the change.
And I don't think build scripts have a public contract anyway )